travis/ipa-init.sh

```diff
@@ -22,7 +22,12 @@
 set -e
 
 # Enable IPA COPR repo
-dnf copr enable -y @freeipa/freeipa-4-7
+# NOTE: IPA does not build for F30
```

Comment:
Not sure how this correlates to the below, but they should definitely enable this. Do you mean `F31`?

Reply:
The `freeipa-master-nightly` [COPR project](https://copr.fedorainfracloud.org/coprs/g/freeipa/freeipa-master-nightly/) does not include F30. So, you will see an error if you try to enable this repo in F30